code/src/components/GameScreen.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import labyrinth, { generateGame } from "reducers/labyrinth";
+import { 
+    CardContainer,
+    Content,
+    SecondaryText, 
+    Action 
+} from "./Styles";
+
+
+const GameScreen = () => {
+    const stage = useSelector(store => store.labyrinth.stage);
+
+    const dispatch = useDispatch();
+
+    const onActionSet = (action) => {
+        dispatch(generateGame(action));
+    }
+
+    const onQuoteRevert = () => {

```

Comment:
A small reminder about naming a variable that fits better to what the function is all about.